website/content/docs/agent/options.mdx

```diff
@@ -1396,13 +1396,17 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
       certificate will be requested by a proxy before this limit is reached. This
       is also the effective limit on how long a server outage can last (with no leader)
       before network connections will start being rejected. Defaults to `72h`.
-      This value cannot be lower than 1 hour or higher than 1 year.
+      This value cannot be lower than 1 hour or higher than 1 year. Must use hour
+      units or smaller, for example, `1d` is not valid but `24h` is.
 
       This value is also used when rotating out old root certificates from
       the cluster. When a root certificate has been inactive (rotated out)
       for more than twice the _current_ `leaf_cert_ttl`, it will be removed
       from the trusted list.
 
+    - `intermediate_cert_ttl` ((#ca_intermediate_cert_ttl)) The time to live (TTL) for the
```

Comment:
@kyhavlov I noticed this wasn't documented. I added this here but there's probably more we should say?